[BLOG] add `stdlib/lapack` blog post (#886)
Quansight/Quansight-website: apps/labs/posts/implement-lapack-routines-in-stdlib-js.md

---

> title: "Implement LAPACK routines in stdlib-js"

Reviewer: ```suggestion
title: "Implement LAPACK routines in stdlib"
```

The official project name is `stdlib` and not `stdlib.js` or `stdlib-js`.

> authors: [Pranavchiku]

Author: ```suggestion
authors: [pranav-goswami]
```

published: September 30, 2024

> description: "Implement various LAPACK routines in standard of javascript for numerical computing"

Reviewer: ```suggestion
description: "Implementing LAPACK routines for numerical computation in web applications"
```
